lms/djangoapps/certificates/admin.py

```diff
@@ -28,6 +29,11 @@ def __init__(self, *args, **kwargs):
         self.fields['organization_id'] = forms.TypedChoiceField(
             choices=org_choices, required=False, coerce=int, empty_value=None
         )
+        lang_choices = all_languages()
```

Comment:
@BenjiLee I updated the language choices parameter to call this method that translates all the options to the local language
https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/lang_pref/api.py#L65-L75

Do you think this satisfies the concern of openedx admin page users in different languages?

Reply:
Does `all_languages` include language variants like `es-419`? I believe we only want the two-letter code.